.github/actions/import-external-addons/index.js

```diff
@@ -45,12 +45,22 @@ const main = async () => {
         version: externalAddonVersion.version,
         kurlVersionCompatibilityRange: externalAddonVersion.kurlVersionCompatibilityRange,
         origin: externalAddonVersion.url,
+        isPrerelease: externalAddonVersion.isPrerelease || false,
       };
       const existing = findVersion(addonRegistryKurl[addonName], next);
       if(existing) {
-        // At the moment we treat the version as immutable.
-        // In the future we could check the hash and overwrite.
-        info(`Skipping existing addon ${addonBundleName}.`);
+        if (isVersionReleasing(existing, next)) {
+          info(`Releasing addon ${addonBundleName}.`);
+  
+          // only allow changing isPrerelease property for now
+          existing.isPrerelease = false;
```

Comment:
Makes sense that we would only allow changing `isPrerelease`.  Do we want this to be one-way only (true -> false)? Or might is also be useful to allow the other way in case there's a need to undo a release (false -> true)?